### ISSUE TITLE
Output is plain red

### ISSUE THREAD

**Issue by kareem-metwaly** [tuvovan/ANL-HDRI#1]
Hi there,

I downloaded the dataset and the weights. Then, I tested it over the same test set but I got a result of mostly red color, that's an example. Do you know what could cause that? thanks in advance.

![hdr](https://user-images.githubusercontent.com/28205928/94278662-192f0a80-ff19-11ea-9892-9ab6380ec500.jpg)

**Comment by tuvovan** [comment]
Hi,
Thank you for your interest on my work. Could you please let me know the name of the dataset that you have been working on? I will test and may give you the exact answer!

Thanks!

**Comment by kareem-metwaly** [comment]
Thanks for your prompt reply, I'm using Kalantari's dataset as mentioned in the readme file and the same weights file

**Comment by tuvovan** [comment]
hi, 

sorry that was my bad. the result is plain red is because the default path to the test data is wrong.

I've just update the test code.
The default path of weight is "weights/best.h5" and path for testing data is "Test/EXTRA/001/"

So after downloading the weight (ANLNet.h5) and test data, make sure to specify them when you run.

i.e:
python test.py --weight_test_path ANLNet.h5 --test_path Test/EXTRA/001/

Please do the test and let me know if there is any problem ^^

If you find it helpful, kindly give me a star ^^

**Comment by kareem-metwaly** [comment]
Thanks again, yeah I realized that and I already changed that in the code before running it, the result is actually after these modifications, the names were not also consistent with the dataset and I had to change it to work.
Were you able to reproduce the same numerical results as mentioned in the paper? Also for your other repo, NHDRRNet?

**Comment by tuvovan** [comment]
this work is not re-implementation of any paper. just my play around as I'm quite interested in this topic and I'm trying to figure out my own way so this one is just my first step toward it.

NHDRRNet is actually not fully trained and has some modification compared to the original paper. They say that they use the size of 32x32 for the input of global-non local modules even though the input image size is 256x256 (you know after 4 encoder the feature's size is 16x16).

umm, and due to the limitation of my GPU so I have NOT reached 160000 epochs as in the paper.

In summary, I have not tried to get the numerical results and I think the provided weight will not reach the results as specified in the paper because that was not my initial intention and many limitations I had to face :D 

Anyway that is a good start I guess ^^

**Comment by kareem-metwaly** [comment]
Thanks for your help, I was actually interested in an implementation that reproduces the results in that paper, which I agree with you won't be possible with these missing details in the paper.
I am going to close this since it seems the problem is that it requires more training epochs and the provided weights file is in a limited training case
